pkg/collector/adapters/config_to_probe.go

```diff
@@ -23,18 +23,18 @@ import (
 )
 
 var (
-	errNoService    = errors.New("no service available as part of the configuration")
-	errNoExtensions = errors.New("no extensions available as part of the configuration")
+	ErrNoService    = errors.New("no service available as part of the configuration")
```

Comment:
Are you using this outside of the package? It shouldn't be exported without a good reason.

Reply:
Fixed them.